src/base-apis.js

```diff
@@ -1,3 +1,23 @@
+/**
```

Comment:
Paste accident? The method is duplicated above the file header.

Reply:
Oh that's a mistake.